.github/workflows/lint.yml

```
@@ -0,0 +1,12 @@
+name: Renovate Config Linting
+on: [pull_request]
```

Just going to switch this to the syntax we use in other configs for consistency

```suggestion
on:
  pull_request:
```